catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchSource.java

```diff
@@ -117,6 +117,10 @@
 
   private static final String LOCAL_SEARCH_PARAMETER = "local";
 
+  private static final String USERNAME_PROPERTY = "username";
+
+  private static final String PASSWORD_PROPERTY = "password";
```

Comment:
(out of scope for this pr, but for the record) It would be a nice future improvement  to extract out the username/password string as an exported constant so that these changes and the previous change (https://codice.atlassian.net/browse/DDF-2547) wouldn't both have copies of the same two variables.